### PR TITLE
Block Type constants

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**

### CODE REVIEW

File: fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/BlockType.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+package net.fabricmc.fabric.api.util;
+
+import net.minecraft.block.Block;
+
+/**
+ * Block Type Constants.
+ *
+ * @see Block#getBlockType()
+ */
+public class BlockType {
```
Comment (hYdos) [inline fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/BlockType.java:27]: do we really need this?

**Review by hYdos — CHANGES_REQUESTED** [review]
what

File: fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/BlockType.java (hunk 1 above)
Comment (BoogieMonster1O1) [reply fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/BlockType.java:27]: yes we do. random integers in `getBlockType` make no sense just as how random integers in nbt make no sense

Comment (BoogieMonster1O1) [reply fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/BlockType.java:27]: in later versions this is an enum hence fapi doesnt have it

**Review by hYdos — APPROVED** [review]
👍🏻

**Review by BluCobalt — APPROVED** [review]
merg

**Pull request merged**